next/generateOpenApiClient.ts

```diff
@@ -0,0 +1,85 @@
+const { execSync } = require('node:child_process')
+const { readFile, writeFile } = require('node:fs').promises
+const { rimrafSync } = require('rimraf')
+
+/*
+ * This script generates OpenAPI clients for the specified type.
+ *
+ * In addition to using @openapitools/openapi-generator-cli it does these things:
+ * 1. Uses Docker to run the OpenAPI generator CLI to generate the client, it produces consistent results and works
+ *   on all platforms (the official CLI breaks when the schema generator needs update). This one uses the latest release
+ *   automatically.
+ * 2. Removes the existing client directory before generating a new one.
+ * 3. Replaces all `options?: any` with `options?: RawAxiosRequestConfig` in the generated client. This is needed until
+ *   the respective issue is fixed.
+ * 4. Formats the generated client with Prettier.
+ */
+
+/**
+ * Replaces all `options?: any` with `options?: AxiosRequestConfig`.
+ * Needed until https://github.com/OpenAPITools/openapi-generator/issues/15985 is fixed
+ */
```

Comment:
duplicate wit hthe text above (the link can be placed in point 3. and this piece can be removed

Reply:
Done.